src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -49,9 +49,15 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
-      <% if (request.getSession().getAttribute("user") != null) { %>
-    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else { %>
+          <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+        <%if(request.getSession().getAttribute("user").equals("EmilyArroyo")|request.getSession().getAttribute("user").equals("AlexandriaStorm")|request.getSession().getAttribute("user").equals("AnaVitoriadoValleCosta")|request.getSession().getAttribute("user").equals("KevinWorkman")|request.getSession().getAttribute("user").equals("GavinLifrieri")) { %>
```

Comment:
Just double checking, this block of code will display the an admin item in the nav bar if the user logged in is one of the hardcoded users you have put as admins correct?